From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <jkleinsc@electronjs.org>
Date: Tue, 29 Oct 2024 16:12:28 -0400
Subject: disable ELF CREL on Linux Arm

https://chromium-review.googlesource.com/c/chromium/src/+/5938657
enabled ELF CREL, but this causes a Segmentation fault in GetAccessFlags
in page_allocator_internals_posix.cc when running on, so disable
ELF CREL on Linux Arm

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 5d29e8f419048989f1d902d7dbd5e38e94b93251..c3c56ffac051bea5df7702b9adc2641d0d34fcdd 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -618,7 +618,7 @@ config("compiler") {
 
     # Enable ELF CREL (see crbug.com/357878242) for all platforms that use ELF
     # (excluding toolchains that use an older version of LLVM).
-    if (is_linux && !llvm_android_mainline &&
+    if (is_linux && !llvm_android_mainline && current_cpu != "arm" &&
         default_toolchain != "//build/toolchain/cros:target") {
       cflags += [ "-Wa,--crel,--allow-experimental-crel" ]
     }
